packages/core/config/webpack.common.js

```diff
@@ -157,6 +157,7 @@ const getSharedPlugins = (isEnvProduction) => {
     // The define plugin will replace these keys with their values during build
     // time. Note that they're used in packages/web/src/config.ts, and made available in globalThis
     new webpack.DefinePlugin({
+      ['process.env.RWJS_SRC_ROOT']: redwoodPaths.base,
```

Comment:
I've been struggling to get this up and running in a test project, but _in my head_ this should work ;)

It's so that the vscode links can work in the stack trace.

Reply:
@dac09 Do you think `RWJS_SRC_ROOT` should be added as a web config (like the DB Auth path) in global?

Then in toml:

```
srcRoot="/my/path/to/app/src"
```


and

```
global.RWJS_SRC_ROOT = process.env.RWJS_SRC_ROOT as string
```